# `python:3.5.6-alpine3.8`

## Docker Metadata

- Image ID: `sha256:7aca0c162be3cccf5ae6a4ebae403d575b77c1405230fc449de4c4071f969fe0`
- Created: `2018-08-02T23:16:33.951541829Z`
- Virtual Size: ~ 75.29 Mb  
  (total size of all layers on-disk)
- Arch: `linux`/`amd64`
- Command: `["python3"]`
- Environment:
  - `PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG=C.UTF-8`
  - `GPG_KEY=97FC712E4C024BBEA48A61ED3A5CA953F73C700D`
  - `PYTHON_VERSION=3.5.6`
  - `PYTHON_PIP_VERSION=18.0`
